android/examples/run-first-test/first.conf.js

```diff
@@ -9,13 +9,16 @@ exports.config = {
   exclude: [],
 
   capabilities: [{
-    project: "First Webdriverio Android Project",
-    build: 'Webdriverio Android',
-    name: 'first_test',
-    device: 'Google Pixel 3',
-    os_version: "9.0",
-    app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
-    'browserstack.debug': true
+    platformName: "Android",
+    "appium:platformVersion": "9.0",
+    "appium:deviceName": 'Google Pixel 3',
+    "appium:app": process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
```

Comment:
This should not be needed as service will populate the app key (similar to how we don't add local key in caps as that is also filled in by the service based on conf)